feature-libs/user/profile/styles/_new-profile.scss

```diff
@@ -45,4 +38,20 @@
     width: 100%;
     max-width: 35rem;
   }
+
+  .btn-group-row {
+    display: flex;
+    flex: 100%;
+    justify-content: end;
+  }
+
+  .btn-group {
+    display: flex;
+    justify-content: space-between;
+    width: 40%;
+  }
+
+  .ng-select .ng-select-container {
+    background-color: $background ;
```

Comment:
* Please use our `—cx-color` variables for colours and `@include type()` mixin for predefined fonts. We are no longer using Bootstrap variables.
* Please make sure all your CSS code follows the guidelines provided on this wiki page https://wiki.one.int.sap/wiki/display/spar/UI+Styling+and+Accessibility+Onboarding 
* Please reach out to Spartacus Theme Owner if any doubts and to follow up with code review: @morganm58